sample-compose-fragments/src/main/res/layout/fragment_inset_text_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need a separate layout for each fragment when they are all going to be the same?